core/trino-main/src/test/java/io/trino/sql/planner/assertions/ExchangeMatcher.java

```diff
@@ -37,15 +37,15 @@
         implements Matcher
 {
     private final ExchangeNode.Scope scope;
-    private final ExchangeNode.Type type;
```

Comment:
just make it `Optional<ExchangeNode.Type>`. Would that work?

Reply:
changed to Optional